fe/fe-core/src/main/java/com/starrocks/load/pipe/filelist/FileListTableRepo.java

```diff
@@ -0,0 +1,133 @@
+//  Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package com.starrocks.load.pipe.filelist;
+
+import com.starrocks.catalog.CatalogUtils;
+import com.starrocks.load.pipe.PipeFileRecord;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.statistic.StatsConstants;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections4.ListUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+/**
+ * Persist the file-list in an OLAP table
+ */
+public class FileListTableRepo extends FileListRepo {
+
+    private static final Logger LOG = LogManager.getLogger(FileListTableRepo.class);
+
+    protected static final String FILE_LIST_DB_NAME = StatsConstants.STATISTICS_DB_NAME;
+    protected static final String FILE_LIST_TABLE_NAME = "pipe_file_list";
+    protected static final String FILE_LIST_FULL_NAME = FILE_LIST_DB_NAME + "." + FILE_LIST_TABLE_NAME;
+
+    protected static final String FILE_LIST_TABLE_CREATE =
+            "CREATE TABLE IF NOT EXISTS %s (" +
```

Comment:
why not includes file_rows and error fields?

Reply:
I have not implement these two functionalities, so I prefer to postpone adding these two columns